assets/app/scripts/services/storage.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("StorageService", function() {
+    function StorageService() {}
+
+    StorageService.prototype.generateSimpleName = function(prefix, length) {
+      if (!prefix) {
+        prefix = "";
+      }
+      if (!length) {
+        length = 5;
+      }
+      var randomString = Math.round((Math.pow(36, length + 1) - Math.random() * Math.pow(36, length))).toString(36).slice(1);
+      return prefix + randomString;
```

Comment:
This is actually different, it's not a GUID but a 5-letters random that will give you for example `volume-7a91y`. This is how the CLI does it so I'd like to keep the same approach so we have similar id's not matter where the volume were attached. Notice as per @jwforres suggestion we moved this to a filter.